README.md

```diff
@@ -108,160 +111,178 @@ See [CoaL Setup](./docs/developer-guide/coal-setup.md) for a thorough walkthroug
 
     1. Launch VMware at least once after installing VMware.
 
-    2. Download and run the following setup script from this repository to
-       configure VMware's networking.
+    2. Run SDC set up script for VMware:
 
          - Mac:
 
             ```bash
-            sudo ./tools/coal-mac-vmware-setup
+            curl -s https://raw.githubusercontent.com/joyent/sdc/master/tools/coal-mac-vmware-setup | sudo bash
             ```
 
          - Linux:
 
             ```bash
-            sudo ./tools/coal-linux-vmware-setup
+            curl -s https://raw.githubusercontent.com/joyent/sdc/master/tools/coal-linux-vmware-setup | sudo bash
             ```
 
          - Windows:
 
             ```
-            tools\coal-windows-vmware-setup.bat
+            Download https://raw.githubusercontent.com/lloydde/sdc-1/readme-little-less/tools/coal-windows-vmware-setup.bat
+            Run coal-windows-vmware-setup.bat
             ```
 
 4. Unpack the CoaL build that you downloaded in step 1.
+
     - Mac:
 
         ```bash
-        $ tar xvzf coal-latest.tgz
-        x root.password.20140926t231701z
-        x coal-release-20140918-20140927T030204Z-gec168e5-4gb.vmwarevm/
-        x coal-release-20140918-20140927T030204Z-gec168e5-4gb.vmwarevm/zpool.vmdk
-        x coal-release-20140918-20140927T030204Z-gec168e5-4gb.vmwarevm/USB-headnode.vmxf
-        x coal-release-20140918-20140927T030204Z-gec168e5-4gb.vmwarevm/USB-headnode.vmsd
-        x coal-release-20140918-20140927T030204Z-gec168e5-4gb.vmwarevm/4gb.img
+        $ tar -zxvf coal-latest.tgz
```

Comment:
Yeah, I've never got in the habit of the dash, but when googled examples, it came up -- consistency helps people new to nix.

![xkcd tar comic](http://imgs.xkcd.com/comics/tar.png)

hover text: "I don&#39;t know what&#39;s worse--the fact that after 15 years of using tar I still can&#39;t keep the flags straight, or that after 15 years of technological advancement I&#39;m still mucking with tar flags that were 15 years old when I started."